includes/admin/views/html-admin-page-addons.php

```diff
@@ -11,124 +11,148 @@
 	exit;
 }
 
-?>
-<div class="wrap woocommerce wc_addons_wrap">
-	<nav class="nav-tab-wrapper woo-nav-tab-wrapper">
-		<a href="<?php echo esc_url( admin_url( 'admin.php?page=wc-addons' ) ); ?>" class="nav-tab nav-tab-active"><?php esc_html_e( 'Browse Extensions', 'woocommerce' ); ?></a>
-
-		<?php
-			$count_html = WC_Helper_Updater::get_updates_count_html();
-			// translators: Count of updates for WooCommerce.com subscriptions.
-			$menu_title = sprintf( __( 'WooCommerce.com Subscriptions %s', 'woocommerce' ), $count_html );
-		?>
-		<a href="<?php echo esc_url( admin_url( 'admin.php?page=wc-addons&section=helper' ) ); ?>" class="nav-tab"><?php echo wp_kses_post( $menu_title ); ?></a>
-	</nav>
+$current_section_name = __( 'Browse Categories', 'woocommerce' );
 
-	<h1 class="screen-reader-text"><?php esc_html_e( 'WooCommerce Extensions', 'woocommerce' ); ?></h1>
+?>
+<div class="woocommerce wc-addons-wrap">
+	<h1 class="screen-reader-text"><?php esc_html_e( 'Marketplace', 'woocommerce' ); ?></h1>
 
 	<?php if ( $sections ) : ?>
-		<ul class="subsubsub">
-			<?php foreach ( $sections as $section ) : ?>
-				<li>
-					<a
-						class="<?php echo $current_section === $section->slug ? 'current' : ''; ?>"
-						href="<?php echo esc_url( admin_url( 'admin.php?page=wc-addons&section=' . esc_attr( $section->slug ) ) ); ?>">
-						<?php echo esc_html( $section->label ); ?>
-					</a>
-				</li>
-			<?php endforeach; ?>
-		</ul>
-
-		<?php if ( isset( $_GET['search'] ) ) : // phpcs:ignore WordPress.Security.NonceVerification.Recommended ?>
-			<h1 class="search-form-title" >
-				<?php // translators: search keyword. ?>
-				<?php printf( esc_html__( 'Showing search results for: %s', 'woocommerce' ), '<strong>' . esc_html( sanitize_text_field( wp_unslash( $_GET['search'] ) ) ) . '</strong>' ); // phpcs:ignore WordPress.Security.NonceVerification.Recommended ?>
-			</h1>
-		<?php endif; ?>
-
-		<form class="search-form" method="GET">
-			<button type="submit">
-				<span class="dashicons dashicons-search"></span>
-			</button>
+	<div class="marketplace-header">
+		<h1 class="marketplace-header__title"><?php esc_html_e( 'WooCommerce Marketplace', 'woocommerce' ); ?></h1>
+		<p class="marketplace-header__description"><?php esc_html_e( 'Grow your business with hundreds of free and paid WooCommerce extensions.', 'woocommerce' ); ?></p>
+		<form class="marketplace-header__search-form" method="GET">
 			<input
 				type="text"
 				name="search"
-				value="<?php echo esc_attr( isset( $_GET['search'] ) ? sanitize_text_field( wp_unslash( $_GET['search'] ) ) : '' ); // phpcs:ignore WordPress.Security.NonceVerification.Recommended ?>"
-				placeholder="<?php esc_attr_e( 'Enter a search term and press enter', 'woocommerce' ); ?>">
+				value="<?php echo esc_attr( ! empty( $search ) ? sanitize_text_field( wp_unslash( $search ) ) : '' ); ?>"
```

Comment:
I think we would also need to add `$search` variable in `uses` doc comment in file `includes/admin/class-wc-admin-addons.php:730`.

Reply:
Added it now :)